### ISSUE TITLE
H608D reports only 5 segments, while there are 10

### ISSUE THREAD

**Issue by do285** [wez/govee2mqtt#245]
### Govee Device SKU

H608D

### Govee2MQTT Version

2024.07.13-82ddc6e9

### Describe the issue

only half of the strip is addressable: H608D has 50 leds (10 segments 5 leds each) but HASS only sees 5 segments (25 leds).

### Startup Diagnostics

[2024-07-20T22:14:17 INFO  govee::commands::serve] Kurtyna świetlna (12:4E:CD:39:32:35:21:89 H608D)
[2024-07-20T22:14:17 INFO  govee::commands::serve]   LAN API: ip=192.168.178.76
[2024-07-20T22:14:17 INFO  govee::commands::serve]   Platform API: devices.types.light. supports_rgb=true supports_brightness=true
[2024-07-20T22:14:17 INFO  govee::commands::serve]                 color_temp=Some((2000, 9000)) segment_rgb=Some(0..5)
[2024-07-20T22:14:17 INFO  govee::commands::serve]   Undoc: room=None supports_iot=true ble_only=false
[2024-07-20T22:14:17 INFO  govee::commands::serve]   Quirk { sku: "H608D", icon: "mdi:light-bulb", supports_rgb: true, supports_brightness: true, color_temp_range: Some((2000, 9000)), avoid_platform_api: false, ble_only: false, lan_api_capable: true, device_type: Light, platform_temperature_sensor_units: None, platform_humidity_sensor_units: None, iot_api_supported: true, show_as_preset_buttons: None }

**Comment by davbedwards** [comment]
I have the H7021 and I'm missing light entities for 15 segments from my 30 bulb strand. I was surprised to see when I change the effect in Home Assistant on the main light entity all 30 segments change as expected. I think we are experiencing a similar problem.

**Comment by wez** [comment]
We can only report to HASS what govee reports to us. Please raise this with Govee if you want it to be addressed.

**Comment by do285** [comment]
@wez i don't think this is so easy as in govee app you may provide length (string can be cut to length). so i'd say it should assume full string size for given model but with an option to adjust this if user decided to shorten his string.

**Comment by davbedwards** [comment]
I have been working with Govee customer service. The reply I recently received was "I am sorry for the late reply, I got a confirmation from the team that MQTT(cloud API) only supports 15. You can use Local API to control all the blubs." 

Earlier I was told "I got a reply that this issue will be improved when there is a new version released."
